internal/commands/pull/command.go

```diff
@@ -77,7 +100,7 @@ func (cmd *Command) Flags() []flags.Flag {
 		flags.BoolFlag{
 			Value: &cmd.inputs.DryRun,
 			Meta: flags.Meta{
-				Name:      "dry-run",
+				Name:      flags.FlagDryRun,
```

Comment:
same thing here, no need for the `const` (but if we did need it, lets keep it scoped to `commands/app/flags.go`)